sql/core/src/test/scala/org/apache/spark/sql/execution/joins/BroadcastJoinSuite.scala

```diff
@@ -48,7 +48,7 @@ class BroadcastJoinSuite extends QueryTest with SQLTestUtils {
       .setMaster("local-cluster[2,1,1024]")
       .setAppName("testing")
     val sc = new SparkContext(conf)
-    spark = SparkSession.builder.getOrCreate()
```

Comment:
FYI, after #13352 , I proceeded to #13365 ([SPARK-15618][SQL][MLLIB] Use SparkSession.builder.sparkContext if applicable.)
You can fix the above line like the following.
```
- spark = SparkSession.builder().config(sc.getConf).getOrCreate()
+ spark = SparkSession.builder().sparkContext(sc).getOrCreate()
```